vite.config.js

```diff
@@ -0,0 +1,88 @@
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import dns from "dns";
+
+dns.setDefaultResultOrder("verbatim");
+
+export default defineConfig({
+  plugins: [
+    react({
+      babel: {
+        babelrc: true
+      }
+    })
+  ],
+  css: {
+    preprocessorOptions: {
+      scss: {
+        api: "modern-compiler",
+        // TODO: fix deprecations!
+        silenceDeprecations: [
+          "slash-div",
+          "mixed-decls",
+          "import",
+          "color-functions",
+          "global-builtin"
+        ]
+      }
+    }
+  },
+  define: {
+    "import.meta.env.NODE_ENV": JSON.stringify(process.env.NODE_ENV),
+    "import.meta.env.BUILD_ID": JSON.stringify(process.env.BUILD_ID),
+    "import.meta.env.REDUX_DEVTOOLS": JSON.stringify(process.env.REDUX_DEVTOOLS)
+  },
+  clearScreen: false,
+  build: {
+    // minify: false,
```

Comment:
Warum?

Reply:
War/Ist für debug-Zwecke (build-output).
Hab die Kommentare in der vite.config.js mal ein bisschen aufgeräumt.